parking_permits/resolvers.py

```diff
@@ -439,8 +439,9 @@ def resolve_change_address(_, info, address_id, iban=None):
     # as talpa will get the updated price based on new zone when
     # asking permit price for next month
     open_ended_permits = permits.open_ended()
-    open_ended_permits.update(parking_zone=new_zone)
+    open_ended_permits.update(parking_zone=new_zone, address=address)
 
+    permits = ParkingPermit.objects.active().filter(customer=customer)
```

Comment:
I assume this is for refreshing the queryset after the address updates, no? You should probably do this instead:
```py
# Get the updated permits.
permits = permits.all()
```
This way, if we'd need to edit the filters for the permits query, we only need to change it in a single place.

It's even mentioned [in the documentation](https://docs.djangoproject.com/en/3.2/ref/models/querysets/#all):

> When a QuerySet is evaluated, it typically caches its results. If the data in the database might have changed since a QuerySet was evaluated, you can get updated results for the same query by calling all() on a previously evaluated QuerySet.

Reply:
Good point, updated now to use `permits.all()`.